cmd/scaffold.go

```diff
@@ -0,0 +1,94 @@
+// Copyright © 2018 Sighup SRL support@sighup.io
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/sighup-io/furyctl/util"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+func init() {
+	rootCmd.AddCommand(scaffoldCommand)
+}
+
+var scaffoldCommand = &cobra.Command{
+	Use:   "scaffold",
+	Short: "create required folders and files",
+	Long:  "scaffold a fury project creating required folders and files",
+	Run: func(cmd *cobra.Command, args []string) {
+		scaffoldTerraformModules()
```

Comment:
I would also move these functions out of the `cmd` package(into `furyfile` maybe?) as they are not about command line infrastructure, they are about domain logic.